tool/travis.sh

```diff
@@ -23,7 +23,7 @@ dart ../../bin/dartdoc.dart
 cd ../..
 
 # Run the tests.
-dart test/all.dart
+pub run unittest:unittest
```

Comment:
`pub run` is the way. You want to run the binary. There is no good way to get at a package binary – except `pub run`